src/vm/i386/excepcpu.h

```diff
@@ -28,12 +28,37 @@ class Thread;
                               // Actually, the handler getting set is properly registered
 #endif
 
+#ifdef FEATURE_PAL
+
+extern VOID SetSEHRecord(PEXCEPTION_REGISTRATION_RECORD record);
```

Comment:
I wonder - what if we just defined implemented __writefsdword and __readfsdword for FEATURE_PAL and put the implementation to the unixstubs.cpp? Then the change in this file could be minimalized to just adding declaration of the two functions if FEATURE_PAL is defined.
The implementation in unixstubs.cpp would be trivial - just to add CurrentSEHRecord variable (btw, it needs to be thread local) and let the __writefsdword set it and __readfsdword read it.